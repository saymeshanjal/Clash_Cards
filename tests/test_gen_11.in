51
14 12 1
16 7 3
14 14 3
2 9 1
2 31 5
6 37 2
6 35 1
14 14 5
2 30 3
16 27 4
16 33 2
12 36 1
2 11 5
4 23 4
4 15 1
16 43 3
12 10 1
8 42 5
16 21 2
10 8 4
16 28 4
10 49 3
6 43 4
10 27 4
10 26 1
4 19 1
16 10 1
4 14 1
16 19 4
2 16 2
4 3 5
6 41 5
4 41 5
12 17 2
14 27 3
10 4 4
4 36 5
4 7 4
4 35 5
6 49 3
8 28 1
12 6 4
10 21 2
2 38 5
10 28 4
16 8 4
6 26 5
4 39 4
2 46 4
14 14 5
4 5 4