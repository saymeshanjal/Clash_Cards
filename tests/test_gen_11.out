1499 1933 0
1539 11331 0
448 2984 0
1 1 0
0 2 0
5 6 9
10 10 0
8 3424 0
0 2 0
283 10690 1897
254 1063 11553
460 464 0
0 2 0
0 5 1
3 3 0
2820 10050 0
398 526 0
8 62 0
339 1760 10771
0 234 18
312 10715 1843
110 142 0
1 14 5
6 167 79
125 127 0
3 3 0
5573 7297 0
3 3 0
234 11295 1341
0 1 1
0 6 0
1 19 0
0 6 0
27 152 745
563 2869 0
0 246 6
0 6 0
0 5 1
0 6 0
6 14 0
35 35 0
1 855 68
13 34 205
0 2 0
9 169 74
1 11673 1196
1 19 0
0 5 1
0 2 0
8 3424 0
0 6 0
